.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: Test
+on: push
+jobs:
+  test:
+    name: Run tests and send to coveralls
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
```

Comment:
I believe our ruby apps are able to use a standard convention to set the python version in a dotfile that then works with CI and Heroku. It might make this more portable to eventually move to a shared workflow if that was possible in the python projects as well. (I'm suggesting this as a separate ticket, not a blocker on this). (We still need to set it in two places (Gemfile and the dotfile) but it allows for the CI to be version agnostic at least)

Reply:
Shared workflow for this is already on my to-do list (as part of professional learning, so no ticket yet but once that's done as an example we'll make tickets to update all the python repos as maintenance work).